src/common/KVStore/MockKVStore.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef _MOCK_KVSTORE_
+#define _MOCK_KVSTORE_
+
+#include <unordered_map>
+#include "IKVStore.hpp"
+
+namespace snail{
+
+class MockKVStore : public IKVStore{
+public:
+	MockKVStore() = default;
+	virtual ~MockKVStore() = default;
+	MockKVStore_result_type put(const std::string &key, const std::string &value);
+	MockKVStore_result_type put(const std::string &&key, const std::string &&value);
+	MockKVStore_result_type get(const std::string &key);
+	MockKVStore_result_type get(const std::string &&key);
+private:
+	std::unordered_map<std::string, std::string> hashmap_;
```

Comment:
hash_map_,
maybe store_ will be better.

Reply:
Modified.